source/v2-scheduler.rst

```diff
@@ -0,0 +1,139 @@
+* Supports a retry of operations in case of network glitches
+
+This tool can be used to automatically perform a failover due to node failures, service degradation, or maintenance. 
+
+Please log ProxySQL Admin bug reports here: https://jira.percona.com/projects/PSQLADM.
+
+.. note::
+
+    The Percona Scheduler Admin utility has different features than :ref:`v2-config`. You cannot use options from one utility in the other utility. Mixing the options may cause unintended results.
+
+.. rubric:: Prerequisites
+
+The following are the prerequisites for using the Percona Scheduler Admin:
+
+* The mysql client and my_print_defaults utility must be installed on the system.
+    
+* The ProxySQL and Percona XtraDB Cluster should be up and running.
+
+
+.. rubric:: The percona-scheduler-admin usage info
+
+.. sourcecode:: bash
+
+   percona-scheduler-admin [ options ]
+   
+.. seealso:: :ref:`psa-options` for examples of common operations.
+
+You must include at least one options. The options are either independent, do not require other options, or are dependent on other options. 
+
+These options can be run independently and do not require other options:
+
+
+.. list-table:: Independent options 
+   :header-rows: 1
+   
+   * - Option
+     - Description
+
+
+   * - `--adduser`
+     - Adds the Percona XtraDB Cluster application user to the ProxySQL database
+   * - ``--config-file=<config-file>``
+     - Reads login credentials from a configuration file. The command-line options override configuration file values.
+   * - ``--debug``                            
+     - Enables additional debug logging.
+   * - ``--disable``, ``-d`` 
+     - Removes any Percona XtraDB Cluster configurations from ProxySQL.
+     
+   * - ``--disable-updates``
+     - Disables admin updates for ProxySQL cluster for the current operation by setting the values to false. The default setting does not change the admin variable settings.
+   * - ``--enable``, ``-e``
+     - Auto-configures Percona XtraDB Cluster nodes into ProxySQL.
+   * - ``--help``
+     - Displays the help text.
+   * - ``--is-enabled``                      
+     - Checks if the current configuration is enabled in ProxySQL.
+   * - ``--status``
+     - Returns a status report on the current configuration.
+   * - ``--server=<IPADDRESS>:<PORT>``   
+     - Specifies the IP address and port for a single server.
+   * - ``--trace``                            
+     - Enables shell-level tracing for this shell script.
+   * - ``--use-stdin-for-credentials``
+     - Uses stdin to send credential to the MySQL client instead of process substitution. The default disables the option and process substitution is used.
+   * - ``--version``, ``-v``                      
+     - Prints the version info
+
+
+The following options require other options or a specific mode. If you run these options by themselves or with incorrect options causes an error. 
```

Comment:
If you run these options by themselves or with incorrect options causes an error. -> Running these options by themselves or with incorrect options causes an error.

Reply:
Rewritten